_posts/2023-06-01-cockpit-293.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cockpit 293
+author: jelly
+date: '2023-06-01'
+tags: machines, podman
+slug: cockpit-293
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from cockpit-machines 291 and cockpit-podman 70:
+
+
+##  Vsock device support
+
+Virtual socket support enables communication between the host and guest over a socket. Cockpit Machines now has support for setting up such a device.
+
+![vsock-edit]({{ site.baseurl }}/images/293-vsock-device-support.png)
+
+The user can choose to configure a custom identifier, or let have it assigned automatically upon a VM's boot. The identifier is used by the host to uniquely identify vsock of a specific guest.
+
+![vsock-edit-dialog]({{ site.baseurl }}/images/293-vsock-device-support-2.png)
+
+Please note that vsock still requires special vsock-aware software (e.g. socat) to communicate over the socket.
+
+## Podman: Support for pruning unused containers
+
+Cockpit-Podman's latest release features a new "prune" dialog for bulk deletion of stopped and exited containers, similar to `podman container prune`. This dialog provides the option to unselect containers while displaying container names, creation dates, and owner information to help ensure the intended containers are selected for deletion.
```

Comment:
I would avoid latest release, as this will invaldate. Let's say, Cockpit-Podman now offers a new ...

I think the second sentence is too much info for release note.